resources/assets/tconstruct/lang/zh_TW.lang

```diff
@@ -0,0 +1,965 @@
+################
+#    Blocks    ## zh_TW translation, by CLOVERsquare 翻譯者 黒羽真汐
+################
+
+tile.tconstruct.tooltables.craftingstation.name=工作桌
+tile.tconstruct.tooltables.stenciltable.name=模具桌
+tile.tconstruct.tooltables.partbuilder.name=部件加工台
+tile.tconstruct.tooltables.toolstation.name=工具組裝台
+tile.tconstruct.tooltables.patternchest.name=模具箱
+tile.tconstruct.tooltables.patternchest.tooltip=可存放模具或是鑄模
+tile.tconstruct.tooltables.partchest.name=部件箱
+tile.tconstruct.tooltables.partchest.tooltip=可存放大量的工具部件
+tile.tconstruct.toolforge.name=工具鍛造台
+
+tile.tconstruct.soil.grout.name=水泥
+tile.tconstruct.soil.grout.tooltip=我是水泥！
```

Comment:
Wait for seconds. You have this: https://github.com/SlimeKnights/TinkersConstruct/blob/1.7.10/resources/assets/tinker/lang/zh_TW.lang